packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -135,8 +136,12 @@ export function normalize(
 
   delete json.options
   json.writeKey = settings?.apiKey
-  ctx.userAgent = window.navigator.userAgent
 
+  ctx.userAgent = window.navigator.userAgent
+  const userAgentData = await clientHints(analytics.options.highEntropyValues)
```

Comment:
Can we get the `clientHints` once when the plugin loads, then attach them to each event? This is a value that shouldn't change during the lifetime of a page, and would save us from having to make this an async function/make an 'expensive' call for each event.